draft-ietf-quic-http.md

```diff
@@ -362,10 +362,15 @@ a MAX_PUSH_ID frame.
 
 As with server push for HTTP/2, the server initiates a server push by sending a
 PUSH_PROMISE frame that includes request header fields attributed to the
-request. The PUSH_PROMISE frame is sent on a response stream.  Unlike HTTP/2,
-the PUSH_PROMISE does not reference a stream; when a server fulfills a promise,
-the stream that carries the stream headers references the PUSH_PROMISE.  This
-allows a server to fulfill promises in the order that best suits its needs.
+request. The PUSH_PROMISE frame is sent on the client-initiated, bidirectional
+stream that carried the request that generated the push.  This allows the server
+push to be associated with a request.  Ordering of a PUSH_PROMISE in relation to
+certain parts of the response is important (see Section 8.2.1 of {{!RFC7540}}).
+
+Unlike HTTP/2, the PUSH_PROMISE does not reference a stream; when a server
+fulfills a promise, the stream that carries the stream headers references the
+PUSH_PROMISE.  This allows a server to fulfill promises in the order that best
```

Comment:
Minor nit: would it be clearer to say "references the Push ID"? Multiple PUSH_PROMISE can reference same Push ID so a pushed stream is not satisfying any particular promise.